packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/oktatookta/appidpinokta.md

```diff
@@ -0,0 +1,18 @@
+In the **General Settings** section:
+
+* **Name**: Enter the name that you would expect to see on a button, such as **Sign in with MyOktaOrg**.
+* **Client Id**: Paste the client ID that you obtained from the Okta org that represents the Identity Provider in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. 
+* **Client Secret**: Paste the secret that you obtained in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. 
+* **Scopes**: Leave the defaults. These scopes are included when your Okta org makes a request to the other Okta org that represents the Identity Provider.
+
+    By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory. 
+
+In the **Endpoints** section:
+
+Add the following endpoint URIs for the Okta Identity Provider that you are configuring. In the Okta org that represents the Identity Provider, you can find the endpoints in the well-known configuration document (for example, `https://{theOktaIdPOrg}/.well-known/openid-configuration`.
+
+* **Issuer** - The identifier of the Okta Identity Provider. For example, the Okta org where you created the Identity Provider app: `https://{theOktaIdPOrg}`
```

Comment:
The URL doesn't render. Adding a terminal slash fixes it, but I'm also curious why this happens.